src/commands.rs

```diff
@@ -2028,30 +2029,91 @@ fn cmd_commit(ui: &mut Ui, command: &CommandHelper, args: &CommitArgs) -> Result
     Ok(())
 }
 
+fn resolve_multiple_rewriteable_revsets(
+    revisions: &[RevisionArg],
```

Comment:
nit: `revision_args` is probably clearer, especially since we have another `revisions` variable inside the loop